pkg/broker/ingress/handler.go

```diff
@@ -158,10 +160,18 @@ func (h *Handler) ServeHTTP(response nethttp.ResponseWriter, request *nethttp.Re
 	if res := h.decouple.Send(ctx, broker, *event); !cev2.IsACK(res) {
 		h.logger.Error("Error publishing to PubSub", zap.String("broker", broker.String()), zap.Error(res))
 		statusCode = nethttp.StatusInternalServerError
-		if errors.Is(res, ErrNotFound) {
+
+		switch {
+		case errors.Is(res, ErrNotFound):
 			statusCode = nethttp.StatusNotFound
-		} else if errors.Is(res, ErrNotReady) {
+		case errors.Is(res, ErrNotReady):
 			statusCode = nethttp.StatusServiceUnavailable
+		case grpcstatus.Code(res) == grpccode.PermissionDenied:
+			const msg string = "Failed to publish to PubSub because permission denied.\n" +
+			"Please refer to \"Configure the Authentication Mechanism for GCP\" at " +
+			"https://github.com/google/knative-gcp/blob/master/docs/install/install-knative-gcp.md"
```

Comment:
It might be better to point broker installation page(https://github.com/google/knative-gcp/blob/master/docs/install/install-gcp-broker.md) rather than install knative-gcp page (this is for the Control Plane setup)

Reply:
Thanks Grace! Modified accordingly.